polars/polars-io/src/parquet/read_impl.rs

```diff
@@ -92,7 +92,7 @@ fn rg_to_dfs(
     remaining_rows: &mut usize,
     file_metadata: &FileMetaData,
     schema: &ArrowSchema,
-    predicate: Option<Arc<dyn PhysicalIoExpr>>,
+    predicate: &Option<Arc<dyn PhysicalIoExpr>>,
```

Comment:
In polars we prefer the reference inside the option, so I think this should be `Option<&Arc..>`

Reply:
Feels a bit more natural, changed.